schema-registry/authorizer/authorizer-core/src/test/java/com/hortonworks/registries/schemaregistry/authorizer/core/AuthorizerTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2016-2020 Cloudera, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hortonworks.registries.schemaregistry.authorizer.core;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.BlockJUnit4ClassRunner;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+@RunWith(BlockJUnit4ClassRunner.class)
+public class AuthorizerTest {
+
+    @Test
+    public void authorizerResourcesTests() {
+        Authorizer.Resource resource = new Authorizer.SerdeResource();
+        String res = "SerDe{ serDeName='*' }";
```

Comment:
They are important because they are used to form exception messages for AuthorizationException. They helped me to find a couple of minor bugs. So let's leave them.